modules/hitl/src/backend/index.ts

```diff
@@ -39,7 +39,7 @@ const entryPoint: sdk.ModuleEntryPoint = {
   translations: { en, fr },
   definition: {
     name: 'hitl',
-    menuIcon: 'feedback',
+    menuIcon: 'comment',
```

Comment:
I think you should use the same icon as the HITL-Next module here!

Reply:
Good Catch !! I use the HITL-next icon